lib/yard/handlers/ruby/dsl_handler_methods.rb

```diff
@@ -41,8 +41,9 @@ def handle_comments
           end
 
           object = MethodObject.new(namespace, method_name, scope)
+          object = register(object)
```

Comment:
nit pick here but you can probably just call `object = register MethodObject.new(...)`

Reply:
see the commit message for https://github.com/lsegal/yard/pull/1220/commits/94ecc1de793d1046ca5c58cb463aaf6c28f25dc0, but the gist is im pretty sure the stomping was happening in `register`, when we assign the source

Reply:
ping on this response?